Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -344,7 +343,7 @@ - (void)setDelegate:(id<_WKWebAuthenticationPanelDelegate>)delegate
 + (NSArray<NSDictionary *> *)getAllLocalAuthenticatorCredentialsWithRPID:(NSString *)rpID
 {
 #if ENABLE(WEB_AUTHN)
-    return getAllLocalAuthenticatorCredentialsImpl(@(WebCore::LocalAuthenticatorAccessGroup), rpID, nil).autorelease();
+    return getAllLocalAuthenticatorCredentialsImpl(CFSTR("com.apple.webkit.webauthn") /*WebCore::LocalAuthenticatorAccessGroup */, (__bridge CFStringRef)rpID, nullptr).autorelease();
```

Comment:
Should use `bridge_cast()` from `<wtf/cocoa/TypeCastsCocoa.h>` instead for the cast.